src/util/triplea/MapXMLCreator/MapXMLCreator.java

```diff
@@ -0,0 +1,839 @@
+		        break;
+		        case 14: showInfoMessage("To change the units that a territory begins with, click on the territory and use the window that opens to set the territory's units. To do so, use the + and - buttons below each unit's name to change the quantity of each unit in the territory. You can also edit the text in the result textbox directly to change the territory units. Just type the name of each unit you want to add, followed by ':', followed by the unit quantitiy: Example: 'infantry: 1, artillery: 3, armour: 2, fighter: 1'.", "Help For Current Step");
+		        break;
+					case 15:
+						final String[] settings = GameSettingsPanel.s_allSettings.split(", ");
+						StringBuilder sb = new StringBuilder();
+						sb.append("<html><table border=0><tr>");
+						int setting_nr = 1;
+						for (final String setting : settings)
+						{
+							sb.append("<td>" + setting + "</td>");
+							if (setting_nr % 3 == 0)
+								sb.append("</tr><tr>");
+							++setting_nr;
+						}
+						sb.append("</tr></table></html>");
+						showInfoMessage(
+									"Setting Name: The name of the setting that is applied to the map. Examples: Always on AA, Two hit battleship, and Japanese bid.\r\n\r\nValue: The value of the game setting. Examples: true, false, 0, 5, 32.\r\n\r\nEditable: Whether players are allowed to change the value of the setting when the map is being started.\r\n\r\nMin. N. (Optional): The lowest number that the value can be set to when the user sets the game options in TripleA. Only change this if the setting is a number.\r\n\r\nMax. N.(Optional): The highest number that the value can be set to when the user sets the game options in TripleA. Only change this if the setting is a number.\r\n\r\nTo have the program automatically enter some of the commonly used Game Settings, click the 'Auto-Fill' button between the Back and Next buttons.",
+									"Help For Current Step");
+		        break;
+		        case 16: showInfoMessage("The map notes can be entered in the textbox. You can enter plain text or html code. Then just click the 'Save Map To File' button, and save the file in the map's 'games' folder. (Create the 'games' folder if not already created)", "Help For Current Step");
+		        break;
+				}
+			}
+		});
+		southLeftPanel.add(m_bHelp);
+		
+		southCenterPanel = new JPanel();
+		m_southPanel.add(southCenterPanel);
+		
+		m_southPanel.add(Box.createHorizontalGlue());
+		
+		m_bBack = new JButton("Back");
+		m_bBack.setMnemonic(KeyEvent.VK_B);
+		m_bBack.setFont(new Font("Tahoma", Font.PLAIN, 11));
+		m_bBack.addActionListener(new AbstractAction("Previous Step")
+		{		
+			private static final long serialVersionUID = -8623081809358073307L;
+
+			public void actionPerformed(final ActionEvent e)
+			{
+				SwingUtilities.invokeLater(new Runnable()
+				{
+					public void run()
+					{
+						if (m_currentStep > 1)
+							goToStep(m_currentStep-1);

```

Comment:
Please always use curly braces on if statements.

Second, please do not use an int value to represent the step. If the numbering changes, or if there is a new step, then all the code needs to be reworked.

Instead use a "Step" class or enum, and give it a method "getPreviousStep."  This may be a more difficult change for you, please look at ways where you can avoid the int value if you can.